main/src/http/http_service.rs

```diff
@@ -95,6 +96,7 @@ impl HttpService {
         dbms: DBMSRef,
         coord: CoordinatorRef,
         addr: SocketAddr,
+        host: String,
```

Comment:
Can SocketAddr be used directly? What is the reason why it cannot be used directly？

Reply:
the default socketaddr is 0.0.0.0,